.github/workflows/RunIssueCopilot.yml

```diff
@@ -0,0 +1,15 @@
+name: Run GitHub issue copilot
+on:
+  issues:
+    types: [opened, edited, closed]
+
+jobs:
+  Issue-copilot:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Run GitHub issue copilot
+        uses: MoChilia/IssueCopilot@768eb62c2889b4ad5932342e99ae8300849f7593
```

Comment:
Can we point this to a branch? So we can always test the latest features in our own repo.

Reply:
Ok, then we should treat the main branch carefully. I will also create a build pipeline for it to make sure the latest features are always executable on the main branch.

Reply:
After discussion, we decided to point the action to `v0` in this workflow and release the minor version for the action in the test stage.